view/theme/frio/js/theme.js

```diff
@@ -80,6 +80,20 @@ $(document).ready(function () {
 		target: ".flex-target",
 	});
 
+	// add mention-link button to the second navbar
+	let $mentionButton = $("#mention-link-button");
+	if ($mentionButton.length) {
+		$mentionButton.appendTo("#topbar-second > .container > #navbar-button").css("padding", "2");
```

Comment:
Please don't apply CSS rules using Javascript, it's harder to track than if it's defined in stylesheets. You can add/remove classes, but not change individual CSS rules.

Reply:
Ok.

Reply:
I removed the css from the js and added a margin-top to style.css to class btn. 

It looks now like this
![grafik](https://github.com/friendica/friendica/assets/5894829/4a67c99b-94fc-49ec-b18f-4739cfd86ffe)

instead of the old style

![grafik](https://github.com/friendica/friendica/assets/5894829/8ee164d5-b3a6-4d97-bef7-5ba7226de499)